Makefile.core.mk

```diff
@@ -140,7 +140,10 @@ netlify: netlify_install
 update_ref_docs:
 	@scripts/grab_reference_docs.sh $(SOURCE_BRANCH_NAME)
 
-update_all: update_ref_docs update_examples
```

Comment:
Do we not need update_examples any more?

Reply:
There is no target, it was removed in an earlier PR (noted above). And I removed the script called in that target.